### DISCUSSION TITLE
About new example `virtual_file_system`

### DISCUSSION THREAD

**Discussion by jinzhongjia** [webui-dev/webui#451] [Q&A] [answered]
I don't quite understand what it means. Does this mean that we can directly generate header files through py to embed files?
@hassandraga

**Comment by hassandraga** [comment, marked as answer]
Yes, but this works only for C/C++ for now.
This is to show an example of how to embed files into the final compiled program.
I still thinking how to make this works for all wrappers... I'm still searching.

**Reply by hassandraga** [reply]
More info: https://github.com/webui-dev/webui/blob/3f683ea036deae1a84c7bfb8e4ae8f9c2312dc14/examples/C/virtual_file_system/main.c#L20
